src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -150,11 +157,36 @@ public static function register(?self $handler = null, bool $replace = true): se
             $handler->setExceptionHandler($prev ?? [$handler, 'renderException']);
         }
 
+        self::$initialExceptionHandler = $prev ?? null;
+
         $handler->throwAt(\E_ALL & $handler->thrownErrors, true);
 
         return $handler;
     }
 
+    public static function unregister(): void
+    {
+        restore_error_handler();
+        if (self::$initialErrorHandler) {
+            set_error_handler(self::$initialErrorHandler);
```

Comment:
This should not be done. We never remove the initial error handler from the stack. There is no reason to register it again here, as restoring the error handler already made this initial error handler the active one on the stack. We don't want to have it twice in the stack.

Reply:
the signature of `register()` having this `$replace` argument might actually make things more complex.